status-monitor.lic

```diff
@@ -119,7 +119,7 @@ class GameFilter
     echo("\a")
     pause 0.25
     command_check(line)
-    fput @responses.first
+    fput @responses.first if get_settings.status_monitor_respond
```

Comment:
I'd prefer to see the setting stored into an instance var in `initialize`. What do you think?